service/discovery/azure/azure.go

```diff
@@ -505,7 +505,7 @@ func resourceGroupID(ID *string) voc.ResourceID {
 		return ""
 	}
 
-	id := strings.Join(s[:5], "/")
+	id := strings.ToLower(strings.Join(s[:5], "/"))
```

Comment:
You also could use the function resourceID() for the string in line 501.
`s := strings.Split(resourceID(ID), "/")`

Reply:
That's right, i think 👍 I just used the code of main